pytest/tests/loadtest/locust/common/base.py

```diff
@@ -331,6 +342,18 @@ def __init__(
         super().__init__(message)
 
 
+class InvalidNonceError(RpcError):
+
+    def __init__(
+        self,
+        used_nonce,
+        ak_nonce,
+    ):
+        super().__init__(
+            f"Tried to use nonce {used_nonce} but AK nonce is {ak_nonce}")
```

Comment:
If "AK nonce" is not a common terminology, might be worth unpacking it as "access key nonce"